evap/staff/views.py

```diff
@@ -1893,6 +1898,16 @@ def text_answer_warnings_index(request):
 
 @manager_required
 def user_index(request):
+    form = UserEditSelectionForm(request.POST or None)
+
+    if form.is_valid():
+        user = form.cleaned_data['user']
+        return redirect('staff:user_edit', user.id, request.path)
```

Comment:
Usually, when we want to store a "next"-location, we pass it as a parameter using the [query string](https://en.wikipedia.org/wiki/Query_string) (e.g. when redirecting to the login page). This prevents problems with slashes being part of the value you want to pass.

@janno42 could we live with a static redirect when saving / deleting a user from staff:user_edit?

Reply:
Currently, https://localhost:8000/staff/user/949/edit// throws an error for me because a `reverse` call fails in there because no "next" argument is given. I can imagine that there are a few additional places that link to the user edit page. These would need to be changed with the current solution.

If we really need the "next"-page passed here, we should probably just give it a sane default value, probably the user list or the user index. Then, only the other of the two would need to use that parameter.

Reply:
a static redirect to `staff/user` would be fine